### ISSUE TITLE
nca in PKPDmisc

### ISSUE THREAD

**Issue by krinaj** [dpastoor/PKPDmisc#28]
Hi Devin,

Has nca function changed in PKPDmisc? I am getting the following error message that design need to be specified.

Thanks,
Krina

![image](https://cloud.githubusercontent.com/assets/21271517/19411383/835976c2-92b5-11e6-93a3-c00554333f5d.png)

**Comment by dpastoor** [comment]
I'm guessing that you are also using a package with a similarly named function, probably pknca? This error should not present at any point from my function. 

Try doing PKPDmisc::nca(...) and it should work, this forces the PKPDmisc version. Also, if you show me your devtools::session_info() I can likely tell you the conflict, so you can check the load order of the packages in the future to make sure the PKPDmisc version takes preferences.


Finally, out of curiosity, what do you use the nca function for? It was one of the first functions I ever added and is reasonably clunky and slow (compared to my auc_partial function for example). I haven't really had much of a need of it myself so haven't given it much love, but if I know some use cases I can improve on its capabilities and performance.

**Comment by krinaj** [comment]
Thanks, Devin.

Yes, I was using a similar package "PK". I suspected that the error message
was coming from that package. Therefore, I had cleaned the Environment,
restarted R, and  reloaded R script without that package, but message
continued and therefore I posted question. But I did not think of
PKPDmisc::nca(...).
:) Now, I know. I will give this a try. This morning, I finally ended up
using new PKNCA package to perform nca and it worked well.

I sometime use nca function for quick nca estimates of all major PK
parameter without quitting R. It is easy to use and takes few minutes.

Thanks,
Krina
